lib/xenops_server.ml

```diff
@@ -2186,8 +2181,7 @@ module VM = struct
 					end else
 						Some (queue_operation dbg vm_id op)
 				in
-				Opt.iter (fun t -> t |> Xenops_client.wait_for_task dbg |> ignore) task;
-				Xenops_migrate.(Handshake.send transferred_fd Handshake.Success)
```

Comment:
It's not needed anymore due to the parallel construction of `VM_save` and `VM_save_vgpu`, which synchronises both operations. So the handshake at "Synchronisation point 2" is all we need now.